static/js/commentDataManager.js

```diff
@@ -293,7 +295,13 @@ commentDataManager.prototype._getRepliesStillOnTextSortedByDate = function(comme
     return commentData.replies[replyId].timestamp;
   });
 
-  return _(commentData.replies).pick(sortedReplyIds);
+  // create new objects for each reply. This way we don't keep the objects
+  // references and the api can know if the object changed before send data
+  // to manager.
```

Comment:
agora eu lembro bem quem é quem, onde foi o problema, mas para esquecer é fácil rs. A gente deveria falar que objeto é esse. Pense na treta que foi, um objeto mudou o valor pq o js usa a referência do objeto - esses bugs são difíceis demais para achar.
O "objeto" está em outro arquivo anyway.
Sugestão:

```
we use `lastCommentsDataSent` to keep track of the comments changes, avoiding 
calling the API to update the comment on the manager. 
On the function triggerDataChanged on api.js we have
"this.lastCommentsDataSent = commentsData;"

The main issue is that the object is a reference of commentsData, so any 
change on "commentsData" will reflect on it as well. 
To avoid this misbehavior, we create copies of the replies
"replies[replyId] = Object.assign({}, commentData.replies[replyId]);"
So, any change on the reply object will create a new object different from the one
referenced by this.lastCommentsDataSent.
```